### DISCUSSION TITLE
Releases: Docker Hub

### DISCUSSION THREAD

**Discussion by jdsika** [lichtblick-suite/lichtblick#185] [Ideas]
Will releases be published on Docker Hub ? https://hub.docker.com/

**Comment by aneuwald-ctw** [comment]
Hello @jdsika, the plan is to publish here in GHCR. I will close this topic as we will make it work soon.

**Comment by aneuwald-ctw** [comment]
> Will releases be published on Docker Hub ? https://hub.docker.com/

Forgot to mention but it is already published on GHCR, you can check it here: https://github.com/lichtblick-suite/lichtblick/pkgs/container/lichtblick
